hardhat.config.ts

```diff
@@ -44,56 +45,53 @@ export default {
-          "ConfiguratorLogic": "0xe24eA68C46fe765E030Bc14baFa087a865d389a6"
-        },
-        "@aave/core-v3/contracts/protocol/libraries/logic/PoolLogic.sol": {
-          "PoolLogic": "0x670eFBdFa365A8de51a72447837226852E53fe43"
-        },
-        "@aave/core-v3/contracts/protocol/libraries/logic/BridgeLogic.sol": {
-          "BridgeLogic": "0x10079a3854039F10E071bB3f054D463568EAC317"
-        },
-        "@aave/core-v3/contracts/protocol/libraries/logic/SupplyLogic.sol": {
-          "SupplyLogic": "0x0c950296714AB383ff4D478262863204a5d4788e"
-        },
-        "@aave/core-v3/contracts/protocol/libraries/logic/BorrowLogic.sol": {
-          "BorrowLogic": "0xF65f7A0F52B921e18Ff8e25f14AEAB3978f8fBdB"
-        },
-        "@aave/core-v3/contracts/protocol/libraries/logic/FlashLoanLogic.sol": {
-          "FlashLoanLogic": "0x6B6025aCFfbc5306FA61cB8708b589baBCb7658e"
-        },
-        "@aave/core-v3/contracts/protocol/libraries/logic/EModeLogic.sol": {
-          "EModeLogic": "0x9cfD416c110A3CE6E7c40AB63965EA0a5553aa7c"
-        },
-        "@aave/core-v3/contracts/protocol/libraries/logic/LiquidationLogic.sol": {
-          "LiquidationLogic": "0xC25F04991577522ef546218606E8a35a565bCDA2"
-        }
-      },
+            "@aave/core-v3/contracts/protocol/libraries/logic/BridgeLogic.sol": {
+              "BridgeLogic": "0x9c1a3d7C98dBF89c7f5d167F2219C29c2fe775A7"
+            },
+            "@aave/core-v3/contracts/protocol/libraries/logic/SupplyLogic.sol": {
+              "SupplyLogic": "0xCeAB1fc2693930bbad33024D270598c620D7A52B"
+            },
+            "@aave/core-v3/contracts/protocol/libraries/logic/BorrowLogic.sol": {
+              "BorrowLogic": "0x99E12239CBf8112fBB3f7Fd473d0558031abcbb5"
+            },
+            "@aave/core-v3/contracts/protocol/libraries/logic/FlashLoanLogic.sol": {
+              "FlashLoanLogic": "0xaAF5f437fB0524492886fbA64D703df15BF619AE"
+            },
+            "@aave/core-v3/contracts/protocol/libraries/logic/LiquidationLogic.sol": {
+              "LiquidationLogic": "0x23b13d016E973C9915c6252271fF06cCA2098885"
+            },
+            "@aave/core-v3/contracts/protocol/libraries/logic/EModeLogic.sol": {
+              "EModeLogic": "0x35938C70af13d0c3bBb4e852A9Ab10B20797AeD5"
+            },
+            "@aave/core-v3/contracts/protocol/libraries/logic/PoolLogic.sol": {
+              "PoolLogic": "0x04FaEd9dCb8d7731d89fe94eb3cc8a29E0e10204"
+            },
+            "@aave/core-v3/contracts/protocol/libraries/logic/ConfiguratorLogic.sol": {
+              "ConfiguratorLogic": "0x55C9400Ef6e7779433Dd4c5a0Cdb9514E5f43f96"
+            }
+          },
     },
   },
   solidity: {
     compilers: [
       {
-        version: "0.8.10",
-        settings: {
-          optimizer: { enabled: true, runs: 100_000 },
-          evmVersion: "berlin",
-        },
-      },
-      {
-        version: "0.7.5",
+        version: "0.8.20",
```

Comment:
Why the bump from `0.8.10` to `0.8.20`?

Reply:
Our compiler using `0.8.20` allows for recursion when legacy flag is false.